### PR TITLE
Methods

### PR DESCRIPTION
**Pull request by inetamagone**
Trying to find how to reload tableView after dismissing a new task.

### CODE REVIEW

File: To-do-list/Model/TaskManager.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+//
+
+import UIKit
+
+class TaskManager {
+    
+    // MARK: - Variables
+    
+    var titleArray = [String]()
+    var descriptionArray = [String]()
```
Comment (eugenemokhan) [inline To-do-list/Model/TaskManager.swift:15]: let's make a struct Task {
var title: String
var description: String
}

and save tasks: [Task] = []

File: To-do-list/Model/TaskManager.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ 
+    // MARK: - Methods
+    
+    // ViewController
+    func reloadTableView(tableView: UITableView) {
```
Comment (eugenemokhan) [inline To-do-list/Model/TaskManager.swift:20]: How will you work with this function at future?

**Review by eugenemokhan — APPROVED** [review]
right way

File: To-do-list/Model/TaskManager.swift (hunk 1 above)
Comment (inetamagone) [reply To-do-list/Model/TaskManager.swift:15]: Ok, thank You, will add that

File: To-do-list/Model/TaskManager.swift (hunk 2 above)
Comment (inetamagone) [reply To-do-list/Model/TaskManager.swift:20]: I wanted to use it on ViewController after coming back from TaskViewController. For the moment didn't find way to implement it correctly.

**Pull request merged**